examples/racecar_subscriber.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class RacecarSubscriber < ActiveSupport::Subscriber
```

Comment:
Can we call this e.g. `prometheus_instrumentation.rb`? It's not an example of Racecar in general.

Reply:
Actually, I think a real implementation is better than an example.